windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -2,14 +2,29 @@
 # SPDX-License-Identifier: BSD-2-Clause
 ---
 # Description:
-# Install inbox version of WSL
-
 # Install WSL and distribution
-- name: "Install WSL and distribution"
+#
+# Due to known issue for store version of WSL(https://github.com/microsoft/WSL/issues/9231),
+# "Launching WSL from session zero does not currently work (for example from an ssh connection)",
+# and the support of inbox version is removed from Windows build 25267, the test case will install inbox version for
+# build lower than 25267 until the known issue is resolved.
```

Comment:
Need an empty comment line '#' at the end of description.

Reply:
fixed